src/libraries/Common/src/Interop/Windows/BCrypt/AEADBCryptHandles.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Security.Cryptography;
+using System.Threading;
+using Internal.NativeCrypto;
+
+namespace Internal.Cryptography
+{
+    internal static class AEADBCryptHandles
```

Comment:
The big block of capital letters feels a bit off.  `BCryptAuthenticatedEncryptionHandles`, maybe?